Foundations/Components/CustomText.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CustomText.swift
+//  Foundations
+//
+//  Created by José Danilo Silva do Carmo on 30/03/23.
+//
+
+import SwiftUI
+
+struct CustomText: View {
```

Comment:
I'm not sure whether this component is useful at all, can you explain a bit why it is preferred to use it instead of just writing the text as-is?

Is it a good practice for iOS apps?
Is there any way to just override Text() instead?